website/docs/r/event_orchestration_path_resource.html.markdown

```diff
@@ -0,0 +1,220 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_event_orchestration_path"
+sidebar_current: "docs-pagerduty-resource-event-orchestration-path"
+description: |-
+  Creates and manages an orchestration path (router|unrouted) associated with a Global Event Orchestration in PagerDuty.
+---
+
+# pagerduty_event_orchestration_router
+
+An Orchestration Router allows users to create a set of Event Rules. The Router evaluates events sent to this Orchestration against each of its rules, one at a time, and routes the event to a specific Service based on the first rule that matches. If an event doesn't match any rules, it'll be sent to service specified in the catch_all or to the "Unrouted" Orchestration if no service is specified. Rules are defined as attributes of the orchestration path resource.
+
+## Example of configuring Router rules for an Orchestration
+
+```hcl
+  # In this example the user has defined the router with two rules, each routing to a different service
+  # This example assumes services used in the route_to configuration already exists. So it does not show creation of service resource.
+  # The first rule matches only if either of the two conditions are matched
+  # The second rule matches always
+  # The catch_all routes to unrouted path
+  # but using the catch_all to route to another service is allowed.
+  resource "pagerduty_event_orchestration_router" "router" {
+    parent {
+      # id of the Global Event Orchestrartion
+      id = pagerduty_event_orchestration.my_monitor.id
+    }
```

Comment:
I agree, it will be more readable if the team is a flat property. Regarding the parent object, it's an event_orchestration for router/unrouted and a service for a service path. We can have it configured as a `parent` or `event_orchestration` for router/unrouted and `service` for service paths